js/theme.js

```diff
@@ -241,3 +241,37 @@ if (typeof(window) != 'undefined') {
 
 $(".sphx-glr-thumbcontainer").removeAttr("tooltip");
 $("table").removeAttr("border");
+
+function buildSourceType() {
+    // Github links need the word "source" after the type of tutorial, like "beginner_source"
+    // The tutorial type appears after "tutorials/", like "tutorials/beginner"
+
+    var tutorialType = $("#tutorial-type")[0].innerText;
+
+    // Then "tutorials/" is sliced off, "_source" is added, and this value is returned
+    return tutorialType.split('/')[0] + "_source/";
+}
+
+var downloadNote = $(".sphx-glr-download-link-note.admonition.note");
```

Comment:
Before we start this section of code, let's put in a comment that say something like "this code replaces the default sphinx gallery download buttons with the buttons at the top of the page"